spec/components/requestBodies/PostOrganization.yaml

```diff
@@ -48,7 +49,7 @@ content:
           description: Invoice will use this time zone to display time otherwise UTC will be used. Example "America/New_York"
           type: string
           maxLength: 50
-        reportingCurrency:
+        reportCurrency:
```

Comment:
It seems a bad idea to have a default value for this since it **cannot** be changed, ever.

Reply:
ok, then seems we should to make reportCurrency as required field too and remove default value.

Reply:
@adamaltman so, reportCurrency is required for now